### PR TITLE
Opt in gcp

### PR DESCRIPTION
**Pull request by jabia99**

### CODE REVIEW

**Comment by jabia99** [comment]
Images that will get removed: http://pastebin.test.redhat.com/1088412

File: cloud_cleaner.sh [hunk 1]
```diff
@@ -354,7 +354,8 @@ GCP_PROJECT=$(jq -r '.project_id' "$GOOGLE_APPLICATION_CREDENTIALS")
 $GCP_CMD config set project "$GCP_PROJECT"
 
 # List tagged intances and remove the old enough ones
-INSTANCES=$($GCP_CMD compute instances list --filter='labels.gitlab-ci-test:true' \
+echo -e "------------------\nCleaning instances\n------------------"
+INSTANCES=$($GCP_CMD compute instances list --filter='NOT labels.persist:true AND NOT labels.persist:true' \
```
Comment (jrusz) [inline cloud_cleaner.sh]: Why is it checking twice for `NOT labels.persis:true` ?

File: cloud_cleaner.sh [hunk 2]
```diff
@@ -363,23 +364,32 @@ for instance in ${INSTANCES}; do
...
+                    echo "deleted instance: ${NAME}"
+                fi
         fi
 done
 
 # List tagged images and remove the old enough ones
-IMAGES=$($GCP_CMD compute images list --filter='labels.gitlab-ci-test:true' \
-	| jq -c '.[] | {"name": .name, "creationTimestamp": .creationTimestamp}')
+echo -e "---------------\nCleaning images\n---------------"
+IMAGES=$($GCP_CMD compute images list --filter='NOT labels.persist:true AND NOT labels.persist:true' \
```
Comment (jrusz) [inline cloud_cleaner.sh]: Same question as above.

**Review by jrusz — CHANGES_REQUESTED** [review]
I posted one question in the code. 

Also I'd like to ask if you've thought about the ssh keys that are there, should we perhaps clean them too? I mean the ones from the tests... Idk if they cost money, but it would be easier to see your own key there, add more and might help with provisoning time as it won't have to copy all the keys on each machine every time... 

Otherwise looks good

**Comment by jrusz** [comment]
Forget the keys, they should be removed during the test cleanup rather. I don't see an easy way to delete only obsolete keys and only the ones from tests... So just the comment in the code.

File: cloud_cleaner.sh (hunk 1 above)
Comment (jabia99) [reply cloud_cleaner.sh]: oh wow, my intention was to check `labels` and `tags`, let me fix it real quick

**Review by jrusz — COMMENTED** [review]
I guess images only have labels, right? Looks fine, can you do the dry run once more please? I've tried tagging my image just to see if it's going to stay now.

**Comment by jabia99** [comment]
Sure! Here's the new run: http://pastebin.test.redhat.com/1089041

**Review by jrusz — APPROVED** [review]
LGTM

**Pull request merged**